### PR TITLE
Adds new integration [michaellunzer/Home-Assistant-Custom-Component-Fortnite]

### PR DESCRIPTION
**Pull request by michaellunzer**

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -258,6 +258,7 @@
   "MatthewFlamm/nwsradar",
   "maykar/plex_assistant",
   "Michsior14/ha-laser-egg",
+  "michaellunzer/Home-Assistant-Custom-Component-Fortnite"
```
Comment (KTibow) [inline integration:261]:
```suggestion
  "michaellunzer/Home-Assistant-Custom-Component-Fortnite",
```

**Review by KTibow — CHANGES_REQUESTED** [review]

File: integration [hunk 2]
```diff
@@ -258,6 +258,7 @@
   "MatthewFlamm/nwsradar",
   "maykar/plex_assistant",
   "Michsior14/ha-laser-egg",
+  "michaellunzer/Home-Assistant-Custom-Component-Fortnite",
```
Comment (KTibow) [inline integration:261]:
```suggestion
  "michaellunzer/Home-Assistant-Custom-Component-Fortnite",
  "Michsior14/ha-laser-egg",
```

**Review by KTibow — CHANGES_REQUESTED** [review]

**Review by michaellunzer — COMMENTED** [review]
looks good!

**Comment by michaellunzer** [comment]
Hi @KTibow is there a way to re-run the validation checks on this PR? My pull requests in the brands and wheels repos have been approved and merged. Thanks!

**Comment by ludeeus** [comment]
<https://hacs.xyz/docs/publish/include#re-run-checks>

**Comment by michaellunzer** [comment]
> https://hacs.xyz/docs/publish/include#re-run-checks

thanks @ludeeus!

**Review by UbhiTS — APPROVED** [review]

**Comment by ludeeus** [comment]
You should consider removing all `.DS_Store` files from your repo and push those, then add `.DS_Store` to  `.gitignore`

**Pull request merged**